### ISSUE TITLE
Impossible to upgrade pecl library

### ISSUE THREAD

**Issue by jmkgreen** [example42/puppet-php#90]
There seems to be an undocumented $version parameter which does in itself work, but if you change the value to a new version nothing happens at execution.

Looks like you check if 'pecl info <module>' comes back with something sensible without noting the version installed.

To work around we must uninstall the pecl module then re-run the puppet agent.

**Comment by alvagante** [comment]
Yes, the current code doesn't manage versions upgrades directly.
